plugins/inventory-backend/src/run.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { getRootLogger } from '@backstage/backend-common';
+import { startServer } from './service/server';
+
+startServer({
```

Comment:
should we think about moving https://github.com/spotify/lighthouse-audit-service in as a backend plugin too, since it's mostly useful in this context? and also it'll be really really helpful to have it depend on the inventory service eventually? it would benefit from all yalls tooling and support in a big way.

Reply:
I would personally like that!

The difference though, is that lighthouse is arguably more of a standalone service with actual functionality. We haven't yet decided where the line could be drawn, but it definitely has benefits to keep stuff in the same repo anyway.

I would argue that as long as we have the OPTION to deploy separately as a first class citizen, then we can still recommend people to do so. But those that want to keep things super integrated can also co-deploy with everything else.